lib/vm.js

```diff
@@ -206,13 +258,20 @@ function runInNewContext(code, sandbox, options) {
   options = Object.assign({}, options, {
     [kParsingContext]: sandbox
   });
-  return createScript(code, options).runInNewContext(sandbox, options);
+  return createScript(code, options).runInContext(sandbox, options);
```

Comment:
this should still be `runInNewContext` right?

Reply:
Right, thanks.